docs/getting-started/libnetwork/Demonstration.md

```diff
@@ -51,29 +51,81 @@ The new command `docker network` can be used to create a logical network.
 
-Create three networks.  If you are not running in cloud 
-environment run the following on either host: 
+> Exactly how we create these networks depends on whether we are using the
+> built-in libnetwork IPAM driver or the Calico IPAM driver.  For more details
+> on this, see the documentation for [`calicoctl node`](../../calicoctl/node.md)
+>
+> Furthermore, when running in a cloud environment we need to also set
+> `ipip` and `nat-outgoing` options.
+
+In each case we use the Calico driver `calico` for networking.  This driver is
+run as part of the calico-node in a calico/node-libnetwork container.  For the 
+demonstration, we explicitly choose an IP Pool for each network rather than 
+using the default selections - this is to avoid potential conflicts with the 
+default NAT IP assignment used by VirtualBox.  Depending on your specific 
+environment, you may need to choose different IP Pool CIDRs.
+
+When the default IPAM driver is used, the pool CIDR and `ipip` and 
+`nat-outgoing` configuration are specified as options on the `network create`.
+
+When the Calico IPAM driver `calico` is used, it is necessary to create the 
+Calico IP Pool in advance with the appropriate configuration for `ipip` and 
+`nat-outgoing`.  The Calico IPAM driver runs alongside the Calico network
+driver within the calico/node-libnetwork container.
+
+
+#### Networking using default IPAM in a non-cloud environment
+
+For default IPAM in a non-cloud environment, run: 
 
     docker network create --driver=calico --subnet=192.168.0.0/24 net1
     docker network create --driver=calico --subnet=192.168.1.0/24 net2
     docker network create --driver=calico --subnet=192.168.2.0/24 net3
     
-If you are running in a cloud environment (AWS, DigitalOcean, GCE), you will 
-need to configure the network with `ipip` and `nat-outgoing` options.  On
-either host, run:
+#### Networking using default IPAM in a cloud environment
+
+For default IPAM in a cloud environment (AWS, DigitalOcean, GCE), run:
 
     docker network create --driver=calico --opt nat-outgoing=true --opt ipip=true --subnet=192.168.0.0/24 net1
     docker network create --driver=calico --opt nat-outgoing=true --opt ipip=true --subnet=192.168.1.0/24 net2
     docker network create --driver=calico --opt nat-outgoing=true --opt ipip=true --subnet=192.168.2.0/24 net3
 
-Note that we use the Calico driver `calico`.  This driver is run within 
-the calico-node container.  We explicitly choose an IP Pool for each network
-rather than using the default selections - this is to avoid potential conflicts
-with the default NAT IP assignment used by VirtualBox.  Depending on your
-specific environment, you may need to choose different IP Pool CIDRs.
+#### Networking using Calico IPAM in a non-cloud environment
+
+For Calico IPAM in a non-cloud environment, you need to first create a Calico
+IP Pool with no additional options.  Here we create a pool with CIDR
+192.168.0.0/16.
+
+    calicoctl pool add 192.168.0.0/16
```

Comment:
Yup - as we discussed, adding a pool simply blats over the existing one.  By instructing to just add the pool is a simplification.  We could say that this pool is created by default so you don't need to do anything, but this is simpler.